src/display/xfa_layer.js

```diff
@@ -79,9 +79,14 @@ class XfaLayer {
         break;
       case "select":
         if (storedData.value !== null) {
+          html.setAttribute("value", storedData.value);
           for (const option of element.children) {
             if (option.attributes.value === storedData.value) {
               option.attributes.selected = true;
+            } else {
```

Comment:
nit: `else if` instead of `else { if...`

Reply:
Fixed on commit 07c16d0